lib/common/util/config.js

```diff
@@ -121,7 +121,7 @@ function initializeConfig () {
   const languageId = window.localStorage.getItem('lang')
     ? window.localStorage.getItem('lang')
     : navigator.language
-  const active = languages.find(l => l._id === languageId) ||
+  const active = languages.find(l => l._id.includes(languageId)) ||
```

Comment:
```suggestion
  const active = languages.find(l => languageId.includes(l._id) ||
```
Could we swap it around like this? That way we can set our language string in the language file to just `pl` and then a browser string of `pl` or `pl-PL` will match. right?

Reply:
Although if it works for polish as is, then I'd say we can leave it as is!

Reply:
Yeah I like that, updated to this in https://github.com/ibi-group/datatools-ui/pull/767/commits/f0c2107f1fe807427265d93968cb6339a6cc388f